x-pack/platform/plugins/private/translations/translations/ja-JP.json

```diff
@@ -9641,17 +9641,17 @@
     "xpack.aiAssistant.chatTimeline.messages.system.label": "システム",
     "xpack.aiAssistant.chatTimeline.messages.user.label": "あなた",
     "xpack.aiAssistant.checkingKbAvailability": "ナレッジベースの利用可能性を確認中",
-    "xpack.aiAssistant.conversationList.deleteConversationIconLabel": "削除",
-    "xpack.aiAssistant.conversationList.errorMessage": "の読み込みに失敗しました",
-    "xpack.aiAssistant.conversationList.noConversations": "会話なし",
-    "xpack.aiAssistant.conversationList.dateGroupTitle.today": "今日",
-    "xpack.aiAssistant.conversationList.dateGroupTitle.yesterday": "昨日",
-    "xpack.aiAssistant.conversationList.dateGroupTitle.thisWeek": "今週",
+    "xpack.aiAssistant.conversationList.dateGroupTitle.lastMonth": "先月",
     "xpack.aiAssistant.conversationList.dateGroupTitle.lastWeek": "先週",
+    "xpack.aiAssistant.conversationList.dateGroupTitle.older": "以前",
     "xpack.aiAssistant.conversationList.dateGroupTitle.thisMonth": "今月",
-    "xpack.aiAssistant.conversationList.dateGroupTitle.lastMonth": "先月",
+    "xpack.aiAssistant.conversationList.dateGroupTitle.thisWeek": "今週",
```

Comment:
I'm not sure where this translations come from, it doesn't seem to be releated to your PR. If it's some kind of sorting, please rollback this changes